content/SCALE/SCALEUIReference/Storage/Pools/Datasets/SnapshotsScreens.md

```diff
@@ -0,0 +1,131 @@
+This can result in permanent data loss!
+Do not roll back until all desired data and snapshots are backed up.
+{{< /hint >}}
+{{< expand "Click Here for More Information" "v" >}}
+There are three **Stop Roolback if Snapshot Exists** radio button options that impose safety levels on the rollback operation. 
+When the safety check finds additional snapshots that are directly related to the dataset you are rolling back it cancels the rollback.
+
+![DatasetRollbackFromSnapshotWindow](/images/SCALE/22.12/DatasetRollbackFromSnapshotWindow.png "Dataset Rollback from Snapshot")
+
+| Setting | Description |
+|---------|-------------|
+| **Newer intermeidate, Child, and clone** | Select to stop rollback when the safety check finds any related intermediate, child dataset, or clone snapshots that are newer than the rollback snapshots. |
+| **Newer Clone** | Select to stop rollback when the safety check finds any related clone snapshots that are newer than the rollback snapshot. |
+| **No Safety Check (CAUTION)** | Select to stop rollback if snapshot exists. The rollback destroys any related intermediate, child dataset, and cloned snapshots that are newer than the rollback snapshot.  |
+| **Confirm** | Select to confirm the selection and activate the **Rollback** button. |
+{{< /expand >}}
+
+### Clone Snapshot
+The **Clone To New Dataset** option creates a new dataset that is a copy of the selected snapshot.
+{{< expand "Click Here for More Information" "v" >}}
+
+![CloneToDatasetDialog](/images/SCALE/22.12/CloneToDatasetDialog.png "Clone To New Dataset from Snapshot")
+
+Enter a new name in the **Dataset Name** field or leave the default value if you want to keep track of the dataset based on the snapshot it was created from.
+
+**Clone** creates the new dataset on the **Datasets** screen tree table.
+{{< /expand >}}
+### Delete Snapshot
+The snapshot **Delete** option opens a window that lists the snapshot(s) you select. 
+
+![DeleteSnapshotDialog](/images/SCALE/22.12/DeleteSnapshotDialog.png "Delete Single Snapshot")
+
+**Confirm** activates the **Delete** button.
+
+#### Batch Operations - Delete
+To delete more than one snapshot in one operation, select the checkbox beside the datasets you want to delete and to display the **Batch Operations Delete** option. 
+{{< expand "Click Here for More Information" "v" >}}
+
+![SnapshotsScreenBatachOperations](/images/SCALE/22.12/SnapshotsScreenBatachOperations.png "Batch Operations Delete Snapshot")
+
+**Batch Operations Delete** opens a window listing all selected snapshots.
+
+![DeleteMultipleSnapshotsWindow](/images/SCALE/22.12/DeleteMultipleSnapshotsWindow.png "Batch Operations Delete Snapshot Window")
+
+**Confirm** activates the **Delete** button. If a snaphot has the **[Hold](#snapshot-details-screen)** option selected, an error displays to prevent you from deleting that snapshot.
```

Comment:
```suggestion
**Confirm** activates the **Delete** button. If a snapshot has the **[Hold](#snapshot-details-screen)** option selected, an error displays to prevent you from deleting that snapshot.
```